packages/eslint-config/editor.js

```diff
@@ -64,7 +64,10 @@ module.exports = {
     '@typescript-eslint/no-unsafe-call': 'error',
     '@typescript-eslint/no-unsafe-member-access': 'error',
     '@typescript-eslint/no-unsafe-return': 'error',
-    '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      { argsIgnorePattern: '^_', ignoreRestSiblings: true },
```

Comment:
could you say something about the choice?

Reply:
@hejtful I'm not sure if my editor did this and I committed, or if it was a conscious choice by you?

Reply:
I made this choice. @shn-srl had to add an eslint-ignore comment for [these lines](https://github.com/serlo/frontend/pull/3805/files#diff-bc35fb3fc9546511b24e7c499e87ec4db1eb7be43788c98390327386aefb62a4R53-R54). I thought that rest syntax is a normal part of modern JS, and that unused vars shouldn't stop us from using it.

It's worth mentioning that the changes in the code made the var in those lines used, so there is no need for this change in the config any more. Still, I'd keep this exception, so that we can freely use the rest syntax in the future. I am open to discussing it, of course :D

Reply:
Thank you for the explanation. So just I understand: You want to use `...rest` to exclude fields from an object so you deliberately define the unused variables?

Reply:
Yes.

But, "to exclude fields from an object" is more like "to create a new object without certain fields".

Reply:
thank you